otx/algorithms/action/adapters/mmaction/models/detectors/fast_rcnn.py

```diff
@@ -96,11 +104,13 @@ def patch_pools(self):
         self.roi_head.bbox_head.temporal_pool = ONNXPool3D("temporal", self.roi_head.bbox_head.temporal_pool_type)
         self.roi_head.bbox_head.spatial_pool = ONNXPool3D("spatial", self.roi_head.bbox_head.spatial_pool_type)
 
-    def forward_infer(self, imgs, img_metas):
+    # pylint: disable=no-self-argument
+    @FUNCTION_REWRITER.register_rewriter("mmdet.models.detectors.base.BaseDetector.forward")
```

Comment:
```suggestion
@FUNCTION_REWRITER.register_rewriter("otx.algorithms.action.adapters.mmaction.models.detectors.fast_rcnn.AVAFastRCNN.forward")
```
I believe you tries to patch `AVAFastRCN` only.

This should be a function not a method of `AVAFastRCNN`. Please unindent it.

Reply:
Modified. Thanks!